scripts/vllm-multi-model-chart/README.md

```diff
@@ -0,0 +1,110 @@
+# Deploy vLLM with Multiple Models using Intel® Gaudi® AI accelerators with Kubernetes
+
+This example demonstrates using vLLM to serve multiple models with a single Intel Gaudi HPU from a Kubernetes cluster.
+
+## Prerequisites
+
+To run this example you will need:
+* Docker to build and push the docker image
+* A Kubernetes cluster with Intel® Gaudi® Al accelerators (see the
+  [Kubernetes Installation](https://docs.habana.ai/en/latest/Installation_Guide/Additional_Installation/Kubernetes_Installation/index.html)
+  instructions for Intel Gaudi or the [Intel Kubernetes Service Guide](https://console.cloud.intel.com/docs/guides/k8s_guide.html)
+  for using a Kubernetes cluster from Intel® Tiber™ AI Cloud)
+* [`kubectl`](https://kubernetes.io/docs/tasks/tools/#kubectl) installed and configured
+* [Helm](https://helm.sh/docs/intro/quickstart/) installed and configured
+
+## Docker Image
+
+Before deploying vLLM to Kubernetes, you will first need to build and push a vLLM docker image for HPU. To do this,
+clone the `multi_model` branch from the [HabanaAI fork of vLLM](https://github.com/HabanaAI/vllm-fork/tree/multi_model),
+and build the image using the `Dockerfile.hpu` file. Push the image to a registry that will be accessible by your
+Kubernetes cluster nodes.
+
+```
+# Clone the multi_model branch of the HabanaAI fork of vLLM
+git clone https://github.com/HabanaAI/vllm-fork.git --branch multi_model --single-branch --depth 1
+cd vllm-fork
+
+# Build the docker image
+$ docker build -f Dockerfile.hpu -t vllm-hpu-env .
+
+# Tag and push the image to a container registry
+docker tag vllm-hpu-env <registry/image:tag>
+docker push <registry/image:tag>
+```
+
+### Deploy vLLM using Kubernetes
+
+After the docker image has been built and pushed, use the [Helm](https://helm.sh) chart to deploy vLLM to your cluster.
+
+The Helm chart includes:
+* Secret used for storing a [Hugging Face token](https://huggingface.co/docs/hub/en/security-tokens)
+* [Persistent volume claim (PVC)](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) for the [Hugging Face Hub model cache](https://huggingface.co/docs/huggingface_hub/en/guides/manage-cache) directory
+* Deployment to run the vLLM model server
+* Service to expose the deployment
+
+1. Before installing the Helm chart, update the [`values.yaml` file](values.yaml) to configure your job. Important
+   values that you will need to set are:
+   * Set `image.repository` and `image.tag` based on the docker image that you have built and pushed.
+   * Set`secret.hf_token` with your Hugging Face token, if you are using gated models.
```

Comment:
The `values.yaml` file uses camel case for parameter names, so `hfToken` is what you mean here.